src/main/java/bio/terra/metadata/StudyTable.java

```diff
@@ -3,19 +3,19 @@
 import bio.terra.model.ColumnModel;
 import bio.terra.model.TableModel;
 
-import java.util.ArrayList;
-import java.util.List;
+import java.util.*;
```

Comment:
I think we should not use * imports. In fact, I thought we had checkstyle set to flag those.
You can keep IntelliJ from making them with a setting.

Reply:
changed the java code style so it won't automatically use this with more than 5 individual classes from a package